Formula/ioctl.rb

```diff
@@ -0,0 +1,30 @@
+#------------------------------------------------------------------------------
+# ioctl.rb
+#
+# Homebrew formula for ioctl.  Homebrew (http://brew.sh/) is
+# the de-facto standard package manager for OS X, and this Ruby script
+# contains the metadata used to map command-line user settings used
+# with the 'brew' command onto build options.
+#
+# Copyright (c) 2019 IoTeX Foundation
+#------------------------------------------------------------------------------
+class Ioctl < Formula
+  desc "Iotex ctl is a command-line interface for interacting with the IoTeX blockchain"
+  homepage "https://docs.iotex.io/developer/get-started/ioctl-install.html"
+  url "https://github.com/iotexproject/iotex-core/archive/v1.1.1.tar.gz"
+  sha256 "fcff5ce4231253ebbe04e0405a36a7b89fb2d6c1b30ab0b5deac6ac6f84f9f8b"
+  license "Apache-2.0"
+
+  depends_on "go"
+  depends_on "make"
+
+  def install
+    ENV["GOPROXY"] = "https://goproxy.io"
```

Comment:
```suggestion
``

This shouldn't be needed (if build script doesn't use it in some unusual way)

Reply:
@Madongming does the build script use it in some unusual way?

Reply:
Are there any other issues to deal with? When can pr be merged?

Reply:
When all questions are addressed and not just dismissed.